components/public-api/gitpod/v1/workspace.proto

```diff
@@ -16,7 +17,11 @@ service WorkspaceService {
   // WatchWorkspaceStatus watchs the workspaces status changes
   //
   // workspace_id +return NOT_FOUND Workspace does not exist
-  rpc WatchWorkspaceStatus(WatchWorkspaceStatusRequest) returns (stream WatchWorkspaceStatusResponse) {}
+  rpc WatchWorkspaceStatus(WatchWorkspaceStatusRequest)
```

Comment:
please remove optional keyword from WatchWorkspaceStatusRequest

Reply:
I have a feeling that I removed it already. Could you please rebase and make sure that changes from https://github.com/gitpod-io/gitpod/pull/19038 are not lost. I see again optional keywords here and there.

Reply:
There's no optional keyword, I rebased to main minutes ago, could you pull the latest changes?

Reply:
@mustard-mh I see that changes were lost in other places too, like renaming of item to workspace property. 😢